### PR TITLE
Add default_server for nginx processing and correct http spelling

### PR DESCRIPTION
**Pull request by TylerSheffels**
@dannyroberts 

I still want to figure out how this will work if two servers are specified as default  (e.g. staging and prod in this configuration).

### CODE REVIEW

**Comment by TylerSheffels** [comment]
@czue This has the change we talked about this morning and a couple spelling fixes.

File: ansible/roles/nginx/vars/cchq_ssl.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 nginx_sites:
 - server:
    file_name: "{{ deploy_env }}_commcare"
-   listen: "443 ssl"
+   listen: "443 ssl {{ nginx_set_default_server }}"
```
Comment (czue) [inline ansible/roles/nginx/vars/cchq_ssl.yml]: nitpicky but this variable name seems odd to me. (not sure where the `set` part comes from)

**Comment by czue** [comment]
nothing blocking. the other bugs you found were all in untracked files?

**Comment by TylerSheffels** [comment]
@czue Yeah, there were a couple changes in the untracked configuration, but this was the blocking bug.

**Pull request merged**